Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -167,45 +167,57 @@ def base_dn_verified(base_dn):
     return True
 
 
-def generate_dn_and_remove_from_user_profile(user):
+def generate_unique_cn(default_base_dn, cn):
+
+    changing_cn = cn
+    i = 1
+    while check_if_user_exists_by_attribute(default_base_dn, "cn", changing_cn):
+        changing_cn = cn + str(i)
+        i += 1
+        if i == 30:
+            raise Exception("User CN couldn't be generated")
+    return changing_cn
```

Comment:
is this the solution we have to make pytest pass?

Reply:
No, but I think we should keep it just so we won't have a case of too many calls, wdut?